common/test/views/support/cleaner/AmpEmbedCleanerTest.scala

```diff
@@ -140,76 +156,203 @@ class AmpEmbedCleanerTest extends FlatSpec with Matchers {
 
   /////////////////////////////
   // Soundcloud cleaner
+  // Note: There are two formats of src url that have been encountered, testing has been adapted to check both of these
+  //        formats
   /////////////////////////////
 
- "AmpEmbedCleaner" should "replace an iframe in an audio-element with an amp-soundcloud element" in {
-   val soundcloudUrl = "http://www.soundcloud.com%2Ftracks%2F1234"
-   val doc = s"""<html><body><figure class="element-audio"><iframe src="$soundcloudUrl"></iframe></figure></body></html>"""
+ "AmpEmbedCleaner" should "replace an iframe in an audio-element that has a src url from soundcloud.com which matches the first pattern, with an amp-soundcloud element" in {
+   val doc = <html>
+                <body>
+                  <figure class="element-audio">
+                    <iframe src={soundcloudUrlV1}></iframe>
+                  </figure>
+                </body>
+             </html>.toString()
```

Comment:
Can we extract this into a method? A similar pattern is used in a number of tests